pkgs/top-level/all-packages.nix

```diff
@@ -6635,7 +6635,7 @@ with pkgs;
   freetube = callPackage ../applications/video/freetube { };
 
   freqtweak = callPackage ../applications/audio/freqtweak {
-    wxGTK = wxGTK31-gtk2;
+    wxGTK = wxGTK32;
```

Comment:
Is there any gtk2 stuff we could clean up?

Reply:
Not really:

```
  nativeBuildInputs = [ autoconf automake pkg-config ];
  buildInputs = [ fftwFloat libjack2 libsigcxx libxml2 wxGTK ];
```